bitcoin/src/network/constants.rs

```diff
@@ -119,6 +119,28 @@ impl Network {
             Network::Regtest => "regtest",
         }
     }
+
+    /// Converts a `bitcoind -chain` argument name to its equivalent `Network`.
+    ///
+    /// ```bash,no_run
```

Comment:
We can remove the `,no_run` - that was a mistake on `to_core_arg` by me that I now see I haven't fixed yet. Will fix now.

Reply:
#1430

Reply:
Addressed in [cb5dea4](https://github.com/rust-bitcoin/rust-bitcoin/pull/1428/commits/cb5dea45026e59c8948f149ce46af389c09abd31)